modules/ztp-image-based-upgrade-installing-lifecycle-agent-using-cli.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-lcao-using-cli_{context}"]
+= Installing the {lcao} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {lcao} on both the seed and target cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the {rh-rhacm} Operator 2.9.2 or later.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Subscription` CR:
+.. Define the `Subscription` CR and save the YAML file, for example, `lcao-subscription.yaml`:
++
+[source,yaml]
+----
+# PLACEHOLDER DATA, CHECK Subscription INFO
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-lifecycle-agent-subscription
+  namespace: openshift-operators
+spec:
+  channel: "stable"
+  name: lifecycle-agent-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
```

Comment:
```suggestion
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: openshift-lifecycle-agent-subscription
  namespace: openshift-operators
spec:
  channel: "alpha"
  name: lifecycle-agent
  source: redhat-operators
  sourceNamespace: openshift-marketplace
```

Reply:
updated prtoposed yaml based on what I can see in [upstream as of now](https://github.com/openshift-kni/lifecycle-agent/blob/main/config/manifests/bases/lifecycle-agent.clusterserviceversion.yaml#L182), but PTAL @jc-rh @rauhersu (specially the channel 'alpha')